### ISSUE TITLE
[BUG] CHANGE TITLE

### ISSUE THREAD

**Issue by Muh-Develop** [derekhe/msfs2020-google-map#169]
BEFORE SUBMIT A BUG, PLEASE READ THE FAQ PAGE HERE: https://github.com/derekhe/msfs2020-google-map/wiki/FAQ

**Describe the bug**
When I run The Msfs 2020 nothing happen, the google map tool doesn't seem to connect to the servers and doesn't load maps.

**To Reproduce**
After a while, When MSFS 2020 Completing running a small windows comes up of Security Alert 

**Screenshots**
![security alert](https://user-images.githubusercontent.com/77156333/147105230-57e8acc5-365c-4b12-83d1-d71db368f108.png)
![Capture d’écran 2021-12-22 150552](https://user-images.githubusercontent.com/77156333/147105262-e5d98659-bb44-4be8-a76e-c45d790485fe.png)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - MOD Version: 2.3.5
 - MSFS Version: 1.19.9.0 , MS Store or Stream: Steam

**Additional context**

**Comment by derekhe** [comment]
Yes, please agree the certificate issue. It is because of the fake http certificate.

**Comment by Muh-Develop** [comment]
Even When I hit the YES, The Message dismissed, but the problem remain the same!!!

**Comment by derekhe** [comment]
Did you open the mod BEFORE the MSFS2020?

**Comment by Muh-Develop** [comment]
Yeah!! I did?!!

**Comment by Robbie75VR** [comment]
Very same problem here... it doesn't load aything.
